README.md

```diff
@@ -139,6 +139,43 @@ This variant runs a reproducible build in a [Docker](https://docs.docker.com/) c
 **What's a reproducible build in context of NEAR?**
 Why is it needed? Explanation of these points and a step-by-step tutorial is present at [SourceScan/verification-guide](https://github.com/SourceScan/verification-guide).
 
+**Custom build variant via `--variant <name>`**
+Beyond defaults in your `[package.metadata.near.reproducible_build]`, you can now
+define **named build variants** in your `Cargo.toml` under:
+
```

Comment:
This should state that it supports all the same fields as `[package.metadata.near.reproducible_build]`, 
listing all them (`image`, `image_digest`, `container_build_command`, `passed_env`) and mentioning that every field is optional.